packages/expo-image-picker/ios/MediaHandler.swift

```diff
@@ -255,15 +257,16 @@ internal struct MediaHandler {
     let url = URL(fileURLWithPath: path)
     return url
   }
-  
-  private func buildVideoResult(for videoUrl: URL, withName fileName: String? = nil) -> SelectedMediaResult {
+
+  private func buildVideoResult(for videoUrl: URL, withName fileName: String?, andId assetId: String?) -> SelectedMediaResult {
```

Comment:
Using `and` in the external parameter name is very uncommon 😅 Don't try to make the function selector be a correct sentence, it usually doesn't make much sense.